styles/index.css

```diff
@@ -62,10 +62,60 @@
     height: 134px;  
     width: 104px;
 } */
-  
-button:hover {
+
+button:hover, #addButton:hover {
```

Comment:
This is a bit redundant, but the reason why the add button color doesn't change on hover is because we already have a background color defined for that specific button. 

I think what we can do is move the background color specified on line 21 as a generic styling for all button elements on the page (`button { background-color: #F... }`). This way, we wouldn't need to add `#addButton:hover` here.